recipes/flufl.enum/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: flufl.enum
+  version: "4.1.1"
+
+source:
+  fn: flufl.enum-4.1.tar.gz
+  url: https://pypi.python.org/packages/65/51/aa79e41a6deefab555a7cc26a917363c67c6b7e2e1b43067f3d11b5e1b7e/flufl.enum-4.1.1.tar.gz
```

Comment:
Please change this to the more readable `https://pypi.io/packages/source/f/flufl.enum/flufl.enum-{{ version }}.tar.gz` URL.